packages/context/src/objects/objects.ts

```diff
@@ -124,11 +132,6 @@ export type ObjectInstance = {
     [key: string]: unknown;
 };
 
-export type RegexValidation = {
```

Comment:
If you remove this exported type, technically it would make this a breaking change.  Any reason for removing it?

Reply:
No particular reason. I'll add this back in.